utils/memcheck-daos-client.supp

```diff
@@ -1,42 +0,0 @@
-{
-   access-0
-   Memcheck:Param
-   socketcall.sendto(msg)
-   ...
-   fun:send
-   fun:sock_comm_send_socket.isra.3
-   ...
-   fun:sock_pe_progress_tx_entry
-   fun:sock_pe_progress_tx_ctx
-   fun:sock_pe_progress_thread
-   fun:start_thread
-   fun:clone
-}
```

Comment:
And this one is still used, which is the last of the four that I can see reported.